### PR TITLE
Develop merge

### PR DESCRIPTION
**Pull request by waTeim**
Add recent changes from Develop

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -33,7 +33,7 @@ install_requires =
     django-saml2-auth==2.2.1
     djangorestframework==3.12.2
     drf-spectacular==0.15.1
-    tycho-api==1.14.1
+    tycho-api==1.15.0
```
Comment (joshua-seals) [inline setup.cfg]: Still cool with removing this entirely if you'd like.

File: appstore/appstore/settings/ordrd_settings.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from .base import *   # noqa: F401,F403
+from product.configuration import ProductSettings, ProductColorScheme
+
+APPLICATION_BRAND = "ordrd"
+
+PRODUCT_SETTINGS = ProductSettings(
+    brand="ordrd",
+    title="Ordr D",
```
Comment (joshua-seals) [inline appstore/appstore/settings/ordrd_settings.py:8]: Think title should be "Ordr-D" or "Ordrd"

File: appstore/core/views.py [hunk 3]
```diff
@@ -53,14 +53,12 @@ def get_brand_details(brand):
...
-        "restartr": {
-            "name": "UNC Restarting Research",
-            "logo": "logo.png",
-        },
+        "restartr": { "name": "UNC Restarting Research", "logo": "logo.png", },
         "heal": {"name": "NIH Heal Initiative", "logo": "logo.png"},
         "argus": {"name": "Argus Array", "logo": "logo.png"},
         "eduhelx": {"name": "EduHelx", "logo": "logo.png"},
         "testing": {"name": "Testing", "logo": "logo.png"},
+        "ordrd": {"name": "Ordr D", "logo": "logo.png"},
```
Comment (joshua-seals) [inline appstore/core/views.py:61]: Think this value for name should be "Ordr-D or "Ordrd"?

Comment (pj-linebaugh) [reply appstore/core/views.py:61]: I don't really care, but we should use the same everywhere.  Going without the dash is good.

File: setup.cfg (hunk 1 above)
Comment (pj-linebaugh) [reply setup.cfg]: I don't think I'll miss it.

**Review by pj-linebaugh — APPROVED** [review]

**Review by joshua-seals — APPROVED** [review]

**Comment by pj-linebaugh** [comment]
Let's get this merged.  I was just working with Griffin on final touches for the UI merge of develop->master/main.  Planning to do this merge tomorrow.

**Review by Hoid — APPROVED** [review]

**Pull request merged**